packages/slider/src/mwc-slider.ts

```diff
@@ -0,0 +1,208 @@
+/**
+@license
+Copyright 2018 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import {FormElement, html, property, observer, query, customElement, Adapter, Foundation} from '@material/mwc-base/form-element.js';
+import {classMap} from 'lit-html/directives/classMap.js';
+import {repeat} from 'lit-html/directives/repeat.js';
+import {style} from './mwc-slider-css.js';
+import MDCSliderFoundation from '@material/slider/foundation.js';
+
+const {
```

Comment:
Since these values must also be in the HTML here, I'm not sure of the value of using these constants.